src/Akka.Remote.Hosting/AkkaRemoteHostingExtensions.cs

```diff
@@ -30,7 +35,7 @@ private static AkkaConfigurationBuilder BuildRemoteHocon(this AkkaConfigurationB
         /// <returns>The same <see cref="AkkaConfigurationBuilder"/> instance originally passed in.</returns>
         public static AkkaConfigurationBuilder WithRemoting(this AkkaConfigurationBuilder builder, string hostname, int port, string publicHostname = null, int? publicPort = null)
         {
-            var hoconBuilder = BuildRemoteHocon(builder, hostname, port);
+            var hoconBuilder = BuildRemoteHocon(builder, hostname, port, publicHostname, publicPort);
```

Comment:
This was the fix - forgot to pass in the parameters to the `BuildRemoteHocon` method.